src/main/external-resources/renderers/Roku-TV8.conf

```diff
@@ -0,0 +1,62 @@
+#----------------------------------------------------------------------------
+# Profile for Roku TV running Network Streaming Player 8.
+# See DefaultRenderer.conf for descriptions of all the available options.
+#
+
+RendererName = Roku TV (NSP 8)
+RendererIcon = roku-roku3.png
+
+# ============================================================================
+# This renderer has sent the following string/s:
+#
+# User-Agent: Roku/DVP-8.0 (308.00E04156A)
+# ============================================================================
+#
+
+UserAgentSearch = Roku/DVP-8.
+
+LoadingPriority = 1
+
+TranscodeVideo = MPEGTS-H265-AAC
```

Comment:
Is H.265 a smart choice? It requires a lot of CPU to encode and the only gain vs H.264 is slightly lower bandwidth. It might make sense for those with a very powerful CPU and a very, very slow network, but I think that's a real corner case. Wouldn't H.264 work better for most users?

Reply:
Nadahar, you are right. this is still very tailored to my setup.
AMD-FX 8350 Vishera (8 core) and 16gb ddr3.
iam by no mean an expert with these config files. 
for me it works pretty well, perhaps we can work on refining it together ?

Reply:
@Dm47021 I still think you will have problems encoding H.265 fast enough under some conditions, and the very little bandwidth you save compared to H.264 isn't really something I think you'll notice. The quality of the image won't be better under H.265, I suspect it could be worse in some circumstances.